runtime/drivers/registry.go

```diff
@@ -123,29 +123,20 @@ func (i *Instance) Config() (InstanceConfig, error) {
 		AlertsDefaultStreamingRefreshCron: "*/10 * * * *", // Every 10 minutes
 	}
 
+	// Resolve variables
+	vars := i.ResolveVariables()
+
 	// Backwards compatibility: Use "__materialize_default" as alias for "rill.models.default_materialize".
```

Comment:
Given we have limited projects now. May be we manually update this for existing projects and remove support for `__materialize_default` completely. Otherwise this would always need to be maintained in future.